sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -140,6 +166,7 @@ object SQLExecution {
     } finally {
       executionIdToQueryExecution.remove(executionId)
       sc.setLocalProperty(EXECUTION_ID_KEY, oldExecutionId)
+      unsetRootExecutionId(sc, executionId.toString)
```

Comment:
If we need to define a new method, shall we define it to accept `Long` directly?